tests/core/engine_adapter/test_integration.py

```diff
@@ -856,18 +867,8 @@ def test_ctas(ctx: TestContext):
         table_description = ctx.get_table_comment(table.db, "test_table")
         column_comments = ctx.get_column_comments(table.db, "test_table")
 
-        # Trino on Hive COMMENT permissions are separate from standard SQL object permissions.
```

Comment:
@treysp this didnt seem to be a problem anymore in Trino 452, the column comments were returned as expected